controls/datagrid/theming-and-styles/group-header-styling.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Group Header Styling
+page_title: .NET MAUI DataGrid Documentation - Group Header Styling
+description: Learn how to style the Group Headers in a grouped DataGrid control.
+position: 2
+slug: datagrid-group-header-styling
+---
+
+# Group Header Styling for .NET MAUI DataGrid
+
+Telerik .NET MAUI DataGrid provides a `GroupHeaderStyle` property which defines the appearance of the group headers when the data is grouped.
+
+>To learn more about the grouping functionality of the DataGrid, go to [Grouping Overview]({%slug datagrid-grouping-overview%}) topic.
+
+`GroupHeaderStyle` is of type `DataGridGroupHeaderStyle` and provides the following styling properties:
+
+* `TextColor`(`Color`)&mdash;Defines the color for the text part of the group header.
+* `SearchMatchTextColor`(`Color`)&mdash;Defines the color that is used for the parts of the group header's text that are search matches. For more details on this, go to [Search as You Type]({%slug datagrid-search-as-you-type%}) topic.
+* `AggregatesTextColor`(`Color`)&mdash;Defines the color for the aggregates part of the group header. For more details on this, go to [Aggregates]({%slug datagrid-aggregates%}) topic.
+* `BackgroundColor`(`Color`)&mdash;Defines the color that fills the area within the header of the DataGrid group header.
+* `SearchMatchBackgroundColor`(`Color`)&mdash;Defines the color that fills the area within the border when the cell contains a search-match. For more details on this, go to [Search as You Type]({%slug datagrid-search-as-you-type%}) topic.
+* `GroupLevelIndentation`(`double`)&mdash;Defines the indentation that accumulates for each group level.
+* `BorderColor`(`Color`)&mdash;Defines the color that fills the border region of the group header.
+* `BorderThickness`(`Thickness`)&mdash;Defines the thickness of the border.
+* `ButtonFontAttributes`(`FontAttributes`), `ButtonFontFamily`(`string`), and `ButtonFontSize`(`double`) options&mdash;Define the font options of the expand and collapse symbol.
+
+  By default, the Button of the group header uses an internal symbol font family. To render text when the button is expanded or collapsed, instead of a symbol, set a font family to the `ButtonFontFamily` property and add the text to the `ExpandButtonText` and `CollapseButtonText` properties. 
```

Comment:
```suggestion
## Group Header Button Properties

By default, the Button of the group header uses an internal symbol font family. To render text instead of a symbol when the button is expanded or collapsed, set a font family to the `ButtonFontFamily` property and add the text to the `ExpandButtonText` and `CollapseButtonText` properties.
```